daml-assistant/src/DAML/Assistant/Command.hs

```diff
@@ -46,13 +46,13 @@ dispatch info = subcommand
 commandParser :: [SdkCommandInfo] -> Parser Command
 commandParser cmds | (hidden, visible) <- partition isHidden cmds = asum
     [ subparser -- visible commands
-        $  builtin "version" "Display SDK version" mempty (pure Version <**> helper)
-        <> builtin "install" "Install SDK version" mempty (Install <$> installParser <**> helper)
-        <> builtin "exec" "Execute command with daml environment." forwardOptions
-            (Exec <$> strArgument (metavar "CMD") <*> many (strArgument (metavar "ARGS")))
+        $  builtin "version" "Display DAML SDK version" mempty (pure Version <**> helper)
```

Comment:
For what? Helps to be specific - I guess this is something like "Display all installed DAML SDK versions. If run inside a project, also shows the SDK version that the project is using"

Reply:
It really only displays the SDK version in use in this specific location. (e.g. inside a project it displays the project's SDK version, outside the project it displays the latest version). It's probably not a very useful command, would be better as an option `--version` that shows the current SDK and DAML assistant version.